src/modules/ProductLicense.rb

```diff
@@ -1100,6 +1100,8 @@ def HandleLicenseDialogRet(licenses, base_product, action)
           log.info "License has been declined."
 
           case action
+          when "refuse"
+            ret = :refuse
```

Comment:
Probably "refuse**d**" is a bit better...
```suggestion
            ret = :refused
```

Reply:
Ups, sorry.

I was updated it due to your comments https://github.com/yast/yast-registration/pull/439.

I also changed the "cancel_action", not only the "ret" value. What do you think?

Reply:
I'd keep the cancel action"refuse" (means what the code should do when canceled, it's an action), and have the result "refused" (what the user did, it's the result).

It would nice to also have the "abort"/"aborted" pair but as this would break the backward compatibility we should not change that.